src/main/webapp/app/tasks/task-search/task-search.component.ts

```diff
@@ -0,0 +1,26 @@
+import {Component, EventEmitter, Output} from '@angular/core';
+import {FormControl, FormGroup, Validators} from '@angular/forms';
+
+/**
+ * Search field for tiny tasks.
+ */
+@Component({
+  selector: 'tiny-task-search',
+  templateUrl: './task-search.component.html',
+  styleUrls: ['./task-search.component.scss']
```

Comment:
it's always important to have `changeDetection: ChangeDetectionStrategy.OnPush` wherever possible for performance reasons